ceph.spec.in

```diff
@@ -201,20 +201,31 @@ BuildRequires:  librdkafka-devel
 %if 0%{with make_check}
 BuildRequires:  jq
 BuildRequires:	libuuid-devel
+%if 0%{?rhel} == 7
+BuildRequires:	python%{python3_version_nodots}-bcrypt
+BuildRequires:	python%{python3_version_nodots}-nose
+%else
 BuildRequires:	python%{python3_pkgversion}-bcrypt
 BuildRequires:	python%{python3_pkgversion}-nose
 BuildRequires:	python%{python3_pkgversion}-pecan
+%endif
 BuildRequires:	python%{python3_pkgversion}-requests
+%if 0%{?rhel} == 8
```

Comment:
could you follow the pattern of 
```
%if 0%{?rhel} == 7
# el7 specific stuff
%else
# el8 stuff
%endif
```
instead of using 
```
%if 0%{?rhel} == 8
# el8 stuff
%else
# el7 specific stuff
%endif
```